Makefile

```
@@ -94,9 +96,8 @@ sec:
 clean:
 	@rm -rf $(FILES)
 
-# install tools used for building, tests and  validations
 .PHONY: goget-tools
-goget-tools: goget-ginkgo
+goget-tools: goget-ginkgo ## install tools used for building, tests and  validations
```

nitpick - an extra space.
```suggestion
goget-tools: goget-ginkgo ## install tools used for building, tests and validations
```